core/src/actors/connections_manager.rs

```diff
@@ -85,26 +80,15 @@ impl ConnectionsManager {
         send_get_config_request(self, ctx, ConnectionsManager::process_config);
     }
 
-    /// Method to create a session actor from a TCP stream
-    fn create_session(stream: TcpStream, session_type: SessionType) {
-        // Create a session actor
-        Session::create(move |ctx| {
-            // Get peer address
-            let address = stream.peer_addr().unwrap();
-
-            // Split TCP stream into read and write parts
-            let (r, w) = stream.split();
-
-            // Add stream in session actor from the read part of the tcp stream
-            Session::add_stream(FramedRead::new(r, P2PCodec), ctx);
-
-            // Create the session actor and store in its state the write part of the tcp stream
-            Session::new(
-                address,
-                session_type,
-                SessionStatus::Unconsolidated,
-                FramedWrite::new(w, P2PCodec, ctx),
-            )
+    /// Method to request the creation of a session actor from a TCP stream
+    fn request_session_creation(stream: TcpStream, session_type: SessionType) {
+        // Get sessions manager address
+        let sessions_manager_addr = System::current().registry().get::<SessionsManager>();
+
+        // Send a message to the SessionsManager to request the creation of a session
```

Comment:
I've applied @aesedepece's suggestion. But related to what James says. I think it could be `SessionsManager` if we want to use its "proper noun" because that's the name of the actor's struct. If we were using the "common noun" related to its functionality (managing sessions), then we could be refering to it as `sessions manager`. But I'm open to writing in the comment whatever you think it's best :)

Reply:
I agree @aesedepece - seems the most straight forward to implement consistently. From a reading perspective, it's easier to call out visually. As a common noun it's easy to lose sight of it in the reading. @mmartinbar I think there's more flexibility with what you suggest, but as we create a ton of documentation it could get a bit hard to consistently apply. That's my 2 cents :)